sources/packages/api/src/services/workflow/workflow-actions.service.ts

```diff
@@ -59,6 +59,23 @@ export class WorkflowActionsService {
     }
   }
 
+  /**
+   * delete application assessment.
+   * @param assessmentWorkflowId workflow Id to be deleted.
+   * @returns status code.
+   */
+  async deleteApplicationAssessment(
+    assessmentWorkflowId: string,
+  ): Promise<number> {
```

Comment:
I would recommend returning a Promise<void> in this case. If I am not wrong, if the status is different from 2xx Axios is going to throw an exception, which means that this method will return always a 2xx.

Reply:
yes, it will throw exception,